packages/salesforcedx-vscode-apex/src/commands/forceApexExecute.ts

```diff
@@ -126,6 +126,7 @@ export class ApexLibraryExecuteExecutor extends LibraryCommandletExecutor<
     response: ContinueResponse<ApexExecuteParameters>
   ): Promise<boolean> {
     const connection = await workspaceContext.getConnection();
+    // @ts-ignore
```

Comment:
How come this is ts-ignored?

Reply:
Oh it's just temporary until we bump the version of @salesforce/core across the other repos